### ISSUE TITLE
Noisy Images are produced when using MNIST on ALI

### ISSUE THREAD

**Issue by xtarx** [tatsy/keras-generative#4]
The generator loss keeps increasing , while the Discriminator loss converges to 0, resulting in Noise like images on MNIST training dataset.

**Comment by tatsy** [comment]
Hi @xtarx,
Actually, I have not tested MNIST dataset on any network model. So, I cannot guarantee that the models properly work for it.
Also, the ALI model is often very unstable as far as I tested CelebA dataset. So, I think some parameter tuning is necessary for each dataset. As you can find in the original paper, they use quite different networks for different datasets.
